src/style/home.css

```diff
@@ -6,6 +6,12 @@
 #97BF0F
 
 */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
@byiringiroscar Please remove this code I am not using it . It make some design part bad